fragments/notifications.js

```diff
@@ -1,12 +1,36 @@
 import { gql } from '@apollo/client'
-import { ITEM_FULL_FIELDS } from './items'
+import { ITEM_FULL_FIELDS, POLL_FIELDS } from './items'
 import { INVITE_FIELDS } from './invites'
 import { SUB_FIELDS } from './subs'
 
 export const HAS_NOTIFICATIONS = gql`{ hasNewNotes }`
 
-export const NOTIFICATIONS = gql`
+export const INVOICIFICATION = gql`
   ${ITEM_FULL_FIELDS}
+  ${POLL_FIELDS}
+  fragment InvoicificationFields on Invoicification {
+    id
+    sortTime
+    invoice {
+      id
+      actionState
+      actionType
+      satsRequested
+      item {
+        ...ItemFullFields
+        ...PollFields
+      }
+      itemAct {
+        id
+        act
+        invoiceId
+        invoiceActionState
+      }
+    }
```

Comment:
On retry, item.invoiceId and itemAt.invoiceId get replaced so we need them to have their own fields for the values.